client/RequestBehavior.ts

```diff
@@ -77,14 +77,12 @@ export class PopupRequestBehavior extends RequestBehavior {
         await client.init();
 
         try {
-            const result = await client.call(command, ...args);
-            client.close();
-            popup.close();
-            return result;
+            return await client.callAndPersist(command, ...args);
```

Comment:
Note for reviewers: this is basically where the magic is happening. The new `callAndPersist()` method makes sure that the receiving AccountsManager RpcServer writes the request into the URL after validating it, thus enabling a page refresh in the AccountsManager (after a page refresh, the request would be read from the URL as if it were a redirect request, but a special `returnUrl` parameter signals that the request should be replied to via postMessage).